src/storage/query/QueryBaseProcessor.inl

```diff
@@ -491,15 +491,6 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectEdgeProps(
     int         cnt = 0;
     bool onlyStructure = onlyStructures_[edgeType];
     Getters getters;
-    std::unique_ptr<nebula::algorithm::ReservoirSampling<
-        std::pair<std::unique_ptr<RowReader>, std::string>>> sampler;
-    if (FLAGS_enable_reservoir_sampling) {
-        sampler = std::make_unique<
-            nebula::algorithm::ReservoirSampling<
-                std::pair<std::unique_ptr<RowReader>, std::string>
-            >
-        >(FLAGS_max_edge_returned_per_vertex);
-    }
 
```

Comment:
498  line  need handle ?

Reply:
We need it, or we would not the whole samples.